### DISCUSSION TITLE
Giving blocks

### DISCUSSION THREAD

**Discussion by OzarkYT** [GriefPrevention/GriefPrevention#631] [Q&A] [answered]
I'm very new to using plugins but I wanted to give my friends some lan for my server just incase some people joined and wanted to grief so I got this plugin but I don't understand how I can give them land withought handing them OP if someone could help me please contact me on discord or comment Slept#6969

**Comment by ShaneBeee** [comment]
I recommend watching the intro video, it gives you all the details on how the plugin works for the players:
https://www.youtube.com/watch?v=VDsjXB-BaE0&t=9s

Its insanely simple and very user friendly, even with a default config, your players can claim land and earn more claim blocks, all by themselves, without the need for an admin to help out.

Also this video gives a good amount of info too:
https://www.youtube.com/watch?v=RWekSeMi1OE&feature=player_embedded

**Comment by Jikoo** [comment, marked as answer]
If you're just looking to start everyone off with a higher number of blocks, adjust the value of `GriefPrevention.Claims.InitialBlocks`. More info about claim limit settings [here](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#claim-limits).

If you want to give a specific users claim blocks, you can use `/AdjustBonusClaimBlocks` to give them additional bonus blocks.
